lunes_cms/cms/migrations/0010_user.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.16 on 2023-03-07 16:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Just adding the autogenerated migration won't magically migrate all existing users to the new model.